docs/ref/models/relations.txt

```diff
@@ -73,6 +73,11 @@ Related objects reference
         :ref:`intermediate model <intermediary-manytomany>` instance(s), if
         needed.
 
+        .. versionchanged:: 3.1
+
+            You can use callables as values in the ``through_defaults`` dictionary
+            and they will be evaluated just before adding the model.
```

Comment:
Should we document that they'll be called once and not once per `objs`?

Reply:
Good point, I hadn't thought about that. I'll rephrase those paragraphs to make it more obvious what "evaluated" means.

Reply:
I've (force)pushed an update to try and clarify the situation.